apps/code-examples/src/app/code-examples/indicators/label/basic/label-demo.component.ts

```diff
@@ -1,7 +1,47 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { SkyIndicatorDescriptionType, SkyLabelType } from '@skyux/indicators';
 
 @Component({
   selector: 'app-label-demo',
   templateUrl: './label-demo.component.html',
 })
-export class LabelDemoComponent {}
+export class LabelDemoComponent {
+  @Input()
+  public set daysUntilDue(days: number) {
+    this.#_days = days;
+    this.updateLabelProperties(this.#_submitted, this.#_days);
+  }
+
+  @Input()
+  public set submitted(submitted: boolean) {
+    this.#_submitted = submitted;
+    this.updateLabelProperties(this.#_submitted, this.#_days);
```

Comment:
Since we're in another setter, should we be using the getter for `daysUntilDue` instead of accessing the property directly?